tdm-be/config/development.json

```diff
@@ -1,7 +1,7 @@
 {
     "hosts": {
         "internal": {
-            "host": "{ngrok host}",
+            "host": "37f5-193-54-107-82.ngrok-free.app",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Maybe go back to `{ngrok host}`?